webvtt.go

```diff
@@ -123,7 +131,9 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 		// Empty line
 		case len(line) == 0:
 			// Reset block name
-			blockName = ""
+			if blockName != webvttBlockNameStyle || webVTTStyles == nil || endOfCss(webVTTStyles.WebVTTStyles) {
```

Comment:
Does it still work if there are several CSS selectors ? 🤔 

Assume the following `Style` block:
```
STYLE
::cue(b) {
    color: peachpuff;
}

::cue(a) {
    color: white;
}
```

Does that pick up the second selector? 🤔

Reply:
Your example above violates with WebVTT format, a blank line is a sign of a completely new scope. In youe example above, we will not know when to start to parse other scopes, you can verify with this validator: https://w3c.github.io/webvtt.js/parser.html. If there is several CSS selector, the format should be
```
STYLE
::cue(b) {
    color: peachpuff;
}

STYLE
::cue(a) {
    color: white;
}
```
or 
```
STYLE
::cue(b) {
    color: peachpuff;
}
::cue(a) {
    color: white;
}
```
And my code also work with above cases

Reply:
I will add some more styles into the test input file of webvtt then.

Reply:
I didn't know that 👍  Thanks!